packages/web/src/scss/components/Item/_Item.scss

```diff
@@ -0,0 +1,85 @@
+@use 'sass:map';
+@use 'theme';
+@use '../../settings/cursors';
+@use '../../tools/typography';
+
+.Item {
+    display: grid;
+    grid-template-columns: auto 1fr auto;
+    padding: theme.$padding;
+    border-radius: theme.$border-radius;
+    background-color: theme.$background-default;
+    cursor: pointer;
+
+    @media (hover: hover) {
+        &:hover {
+            text-decoration: none;
+            background-color: theme.$background-hover;
+        }
+
+        &:active {
+            text-decoration: none;
+            background-color: theme.$background-active;
+        }
+    }
+}
+
+.Item__label {
+    @include typography.generate(theme.$label-typography);
+
+    display: inline-block;
+    grid-column: 2;
+    color: theme.$label-color-default;
+}
+
+.Item__helperText {
+    @include typography.generate(theme.$helper-text-typography);
+
+    display: block;
+    grid-column: 2;
+    grid-row: 2;
+    margin-top: theme.$helper-text-margin-top;
+    color: theme.$helper-text-color-default;
+}
+
+.Item__icon {
+    grid-row: 1;
```

Comment:
The icon needs `display: flex` so it's precisely 24 px tall:

![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/50ba35c8-ddf6-4d02-8b6c-c17042a23095)